### DISCUSSION TITLE
Next and Qiankun are used together, resulting in deadlock

### DISCUSSION THREAD

**Discussion by seainhand** [vercel/next.js#23071] [Help]
**What version of Next.js are you using?**

latest

**What version of Node.js are you using?**

v12.13.1

**What browser are you using?**

Chrome

**What operating system are you using?**

macOS

**How are you deploying your application?**

node server.js

**Describe the Bug**

Next and Qiankun are used together, resulting in deadlock

**Expected Behavior**

no Deadlock

**To Reproduce**

https://github.com/seainhand/qiankun-next

**Comment by seainhand** [comment]
hi bro, can I know which version to fix this problem?😁😁😁

**Comment by seainhand** [comment]
Or is there any way I can fix it by myself in my own code? (english is poor)

**Comment by seainhand** [comment]
closed means what? It was solved? or It's not a problem?

**Comment by ilteoood** [comment]
Hi @timneutkens : do you have any news about this?
